src/com/android/calendar/AllInOneActivity.java

```diff
@@ -1390,39 +1388,6 @@ public boolean onQueryTextSubmit(String query) {
         return true;
     }
 
-    //@Override
```

Comment:
Can you comment why this can be removed?

Reply:
it overrides the method in `NavigationView` I assume, but I don't so any difference in the behavior of the app. @xsoh @dschuermann has one of you more insights here?

Reply:
I missed the other implementation:
> One of the most notable change here is the method:
> `onNavigationItemSelected(MenuItem item)`
> The old Navigation Drawer template's definition of this method was:
> `onNavigationItemSelected(int position, long itemId)`
https://stackoverflow.com/q/32944798
Looks like it's fine.